host_core/lib/host_core/providers/builtin/numbergen.ex

```diff
@@ -25,4 +25,12 @@ defmodule HostCore.Providers.Builtin.Numbergen do
     |> Msgpax.pack!()
     |> IO.iodata_to_binary()
   end
+
+  def invoke("NumberGen.RandomBytes", payload) do
```

Comment:
Should there be an upper limit for bytes that is less that U32 max?

Reply:
Bytes is correct because Rust is handling each character due to ascii as 1 byte.